tokio/src/time/driver/handle.rs

```diff
@@ -20,6 +20,14 @@ impl Handle {
     /// # Panics
     ///
     /// This function panics if there is no current timer set.
+    /// It can be triggered when `Builder::enable_time()` or `Builder::enable_all()`
+    /// are not included in the builder.
```

Comment:
Be aware that markdown ignores newlines (unless there's an empty line). Either reflow them together as one paragraph, or make an empty line between the two sections.